docs/_docs/refs/configurations.md

```diff
@@ -98,7 +98,7 @@ Depending on the file size that will be processed, it might be required to incre
     max_allowed_packet = 256M
     
 
-If using a older version of MySQL, there is a [known issue](https://github.com/box/mojito/issues/120) when creating the schema. One workaround is to use `utf8`
+If using a older version of MySQL, there is a [known issue]({{ site.github_url }}/issues/120) when creating the schema. One workaround is to use `utf8`
```

Comment:
@byronantak - I don't think that issue id exists in this fork - might need to be updated

Reply:
@garionpin Sorry. It put the automerge under the assumption that it worked like azure devops which blocks if there is an unresolved conversation. I fixed it in a follow-up PR
https://github.com/pinterest/mojito/pull/181